notebooks/book2/23/flow_2d_mlp.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<p>Can you figure out what is <code>xlabel</code> and <code>ylabel</code> for this plots? I think ylabel should be same for all, so in that case you can have a common ylabel.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
The plots depict sampling from various distributions. So should I add x1 and x2 as the label?